src/openassetio-python/LoggerInterfaceBinding.cpp

```diff
@@ -15,7 +15,7 @@ inline namespace OPENASSETIO_CORE_ABI_VERSION {
 struct PyLoggerInterface : LoggerInterface {
   using LoggerInterface::LoggerInterface;
 
-  void log(const Str& message, Severity severity) const override {
+  void log(Severity severity, const Str& message) const override {
     PYBIND11_OVERRIDE_PURE(void, LoggerInterface, log, message, severity);
```

Comment:
[fix] Surprised this even compiles!  Something to watch out for.
```suggestion
    PYBIND11_OVERRIDE_PURE(void, LoggerInterface, log, severity, message);
```